Automattic/sensei
# WPML: Translate quizzes with questions

Resolves #5841

In this PR we don't address an issue of not working properly question translation: when you translated a question in a lesson, but the question remain containing the original content.

## Proposed Changes
* Create translations (duplicates) for quizzes and questions.
* Updated properties.
* Configure blocks.
* The PR also contains some code style fixes.

## Testing Instructions

* Enable WPML
* Create a course with lessons, quizzes and questions.
* Create a translation for the course.
* Make sure lessons were "translated" and contain quizzes.
* Make sure you see duplicated questions for the target language (copies of original).
* Go to the frontend and try take the test.
* Now go back to WP Admin -> Sensei LMS -> Lessons.
* Create a lesson without assigning it to a course. Add a quiz with a question(s).
* Create a translation for this lesson.
* Make sure you see a translated lesson in the target language, and a duplicate for the question(s).

## New/Updated Hooks

* `sensei_rest_api_lesson_quiz_created` - Fires after a quiz is created via the REST API.
* `sensei_lesson_quiz_created` - Fires after a quiz is created while saving a lesson in a non-block editor.

## Deprecated Code


* `Sensei_WPML` - replaced with `Sensei\WPML\WPML`.

## Pre-Merge Checklist
